tools/generators/ethereum/contract.go.tmpl

```diff
@@ -16,8 +16,8 @@ import (
 
 	"github.com/ipfs/go-log"
 
-	"github.com/keep-network/keep-common/pkg/chain/ethereum/blockcounter"
 	"github.com/keep-network/keep-common/pkg/chain/ethereum/ethutil"
```

Comment:
What is the plan for contract templates? I still see a bunch of imports from `github.com/ethereum/go-ethereum`.

Reply:
It's addressed in the next PR: https://github.com/keep-network/keep-common/pull/66